docs/_config.yml

```diff
@@ -1,8 +1,8 @@
 name: LitElement
-description: A simple base class for creating fast, lightweight web components
+description: A simple base class for creating fast and lightweight web components
```

Comment:
This sentence feels weird to me after switching the comma for "and".

Reply:
The reason that I tried to replace comma with "and" is for better text flow/layout. The original blurb has huge gap and results in unbalanced text flow. 
<img width="719" alt="screen shot 2018-12-13 at 10 13 28 am" src="https://user-images.githubusercontent.com/406636/49958469-d3c65800-febf-11e8-9067-6cc229a0a04c.png">

Can we try tweak the blurb to make the text flow more even like this? 
<img width="623" alt="screen shot 2018-12-13 at 10 13 54 am" src="https://user-images.githubusercontent.com/406636/49958470-d45eee80-febf-11e8-93a6-2824952902c8.png">